smartseq2_single_sample/analysis/check_hisat2/test_HISAT2.ipynb

```diff
@@ -0,0 +1,227 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import os\n",
+    "from os.path import basename\n",
+    "from google.cloud import storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test HISAT2\n",
+    "\n",
+    "-  Check reference build by `hisat2-inspect`\n",
+    "-  Check HISAT2 PE alignemnt\n",
+    "-  Check HISAT2 SE alignment\n",
+    "\n",
+    "First load metadata.json data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(os.path.expanduser('/Users/jishuxu/Documents/metadata.json')) as f:\n",
+    "    metadata=json.load(f)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Print out `outputs` from each task"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'test_hisat2.bamfile_pe': 'gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/test_hisat2/05f09e22-eb55-400f-8095-9dc2bc2e992f/call-test_hisat2_pe/SRR1295364_gencode.bam',\n",
+       " 'test_hisat2.bamfile_se': 'gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/test_hisat2/05f09e22-eb55-400f-8095-9dc2bc2e992f/call-test_hisat2_se/SRR1295364_gencode.bam',\n",
+       " 'test_hisat2.inspectlog': 'gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/test_hisat2/05f09e22-eb55-400f-8095-9dc2bc2e992f/call-test_hisat2_inspect/hisat2_inspect.log',\n",
+       " 'test_hisat2.logFile_pe': 'gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/test_hisat2/05f09e22-eb55-400f-8095-9dc2bc2e992f/call-test_hisat2_pe/SRR1295364_gencode.log',\n",
+       " 'test_hisat2.logFile_se': 'gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/test_hisat2/05f09e22-eb55-400f-8095-9dc2bc2e992f/call-test_hisat2_se/SRR1295364_gencode.log',\n",
+       " 'test_hisat2.metfile_pe': 'gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/test_hisat2/05f09e22-eb55-400f-8095-9dc2bc2e992f/call-test_hisat2_pe/SRR1295364_gencode.hisat2.met.txt',\n",
+       " 'test_hisat2.metfile_se': 'gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/test_hisat2/05f09e22-eb55-400f-8095-9dc2bc2e992f/call-test_hisat2_se/SRR1295364_gencode.hisat2.met.txt'}"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "metadata['outputs']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inspect Reference Build"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = storage.Client()\n",
```

Comment:
`client = storage.Client()` This could be further discussed, since we may want to pass the credentials to the Client to finish the authentication, currently Kubernetes is responsible for doing this for the listener.

Reply:
great.  I will try to use it.